doc/company.texi

```diff
@@ -1223,17 +1226,18 @@ The following sections give a short overview of the commonly used
 backends bundled with Company.  Each section is devoted to one of the
 roughly outlined groups of the backends.
 
-Some of the backends expose several user options for customization.
-To see the list of the user options, we suggest doing one of the
+Some of the backends expose user options for customization; a few of
+these options are introduced below.  For those who would like to fetch
+the full list of a backend's user options, we suggest doing one of the
 following:
 
 @itemize
 @item
 Execute command @code{M-x customize-group @key{RET} <backend-name>}.
 
 @item
-Open the source file of the backend and run @code{M-x occur @key{RET}
-^(defcustom}.
+Open the source file of the backend and run @w{@code{M-x occur
```

Comment:
I would probably suggest `isearch` instead of `M-x occur`.

The latter takes more effort, including jumping across windows.

Reply:
Updated to also mention `isearch`: it's a good one to remind about, but the main idea here is to propose something to "fetch a list of options". I personally find it more comprehensible to see the options of a new for me package listed row by row someplace, rather than only scattered in the source. In other words, seems I'm a strong `occur` proponent 😄 .

Reply:
Sounds good. :-)